src/components/Links.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
+import { FaUserCircle } from 'react-icons/fa';
 import { Link } from 'react-router-dom';
 
 const LinkPage = () => (
   <>
-    <nav className="navigation-bar d-flex">
-      <h1>React/Redux Bookstore</h1>
-      <ul className="d-flex">
-        <li>
-          <Link to="/" exact> Books </Link>
+    <nav className="navigation-bar">
+      <h1 className="main-heading">Bookstore CMS</h1>
+      <ul className="thelinks">
+        <li className="ho">
+          <Link to="/" exact> BOOKS </Link>
         </li>
-        <li>
-          <Link to="/categories"> Categories </Link>
+        <li className="ca">
+          <Link to="/categories"> CATEGORIES </Link>
         </li>
       </ul>
+      <span className="oval">
+        <FaUserCircle />
+      </span>
```

Comment:
- [ ] Kindly, refer to the attached image to work on the navbar to match with the design provided.

- Try to modify the width of the navbar
- Add margin-left to the user-icon to be on the left side.
- Change the font size of the books, categories, and user-icon

--------------------------------Yours

![header-1](https://user-images.githubusercontent.com/55635977/143398673-7633032b-6a9b-4802-b98b-08aee9a1710a.png)

--------------------------------Original

![header-2](https://user-images.githubusercontent.com/55635977/143398682-d8643656-50a1-4d72-9e03-0ef7ad46e0a5.png)